### DISCUSSION TITLE
Keyboard shortcut to show timer

### DISCUSSION THREAD

**Discussion by kevgilmore** [focustimerhq/FocusTimer#545] [Ideas]
I would like to assign something like 'toggle display timer/minimize' to a keyboard shortcut. This will allow me to quickly view the timer without using my mouse.

Thanks

**Comment by kamilprusko** [comment]
It's added in `main` branch / [1.0-alpha](https://github.com/gnome-pomodoro/gnome-pomodoro/discussions/742) under "Main Window / Bring To Focus".  

<img width="950" height="909" alt="Screenshot From 2025-12-21 16-59-06" src="https://github.com/user-attachments/assets/2e9396fc-1df8-4383-8383-2a5e8b6cd901" />
